site/.dockerignore

```diff
@@ -0,0 +1,9 @@
+/*
+!/Dockerfile
+!/package.json
+!/package-lock.json
+!/__sapper__
+/__sapper__/*
+!/__sapper__/build
+!/static
+!/content
```

Comment:
Could we make it this instead? I think it has same result
```
**
!/Dockerfile
!/package.json
!/package-lock.json
!/__sapper__/build/**
!/content
!/static
```

Ignore absolutely everything except these...

Reply:
Built locally and it was the same

Reply:
That doesn't work

`Creating temporary tarball archive of 3 file(s) totalling 235.3 KiB before compression.` - only package.json, package-lock.json, and Dockerfile are included.

It's got something to do with not being able to include `__sapper__/build/...` if all of `__sapper__` was already excluded.

When I was playing with GCR for the first time a couple of weeks ago, I spent a while wading through SO answers about gitignore trying to get this to work. I'm pretty sure there's not a tidier way to specifically include these files.

Reply:
Ah, right! It works for docker alone, but gcloudignore is a bit different.